packages/react-components/react-drawer/src/components/DrawerInline/DrawerInline.types.ts

```diff
@@ -0,0 +1,45 @@
+import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
+import { DrawerBaseTypes } from '../../util/DrawerBase.types';
+
+export type DrawerInlineSlots = {
+  root: Slot<'div'>;
+};
+
+export type DrawerOpenChangeData = {
+  open: boolean;
+};
+
+/**
+ * DrawerInline Props
+ */
+export type DrawerInlineProps = ComponentProps<DrawerInlineSlots> &
+  DrawerBaseTypes & {
+    /**
+     * Controls the open state of the Drawer
+     *
+     * @default false
+     */
+    open?: boolean;
+
+    /**
+     * Default value for the uncontrolled open state of the Drawer.
+     *
+     * @default false
+     */
+    defaultOpen?: boolean;
+
+    /**
```

Comment:
DrawerBaseTypes, also, include `open` and `defaultOpen`

Reply:
Fixed.